db/migrate/20160708095911_add_devise_token_to_user.rb

```diff
@@ -0,0 +1,13 @@
+class AddDeviseTokenToUser < ActiveRecord::Migration
+
+  def change
+    add_column :users, :provider, :string, null: false, default: "email"
+    add_column :users, :uid, :string, null: false, default: ""
+
+    ## Tokens
+    add_column :users, :tokens, :json
```

Comment:
I see a potential problem on the production db with setting the data type to `:json` 
Don't we need to add 
```
execute 'CREATE EXTENSION IF NOT EXISTS hstore'
```
to this migration?